### ISSUE TITLE
Kafka header names are not case sensitive in REST API

### ISSUE THREAD

**Issue by hansut** [mailgun/kafka-pixy#175]
Hello, when we try to add some Kafka headers when producing a message using the REST API, their keys in Kafka don't keep the original case. For example:

"X-Kafka-APP_REQUEST_UUID" gets to Kafka as "App_request_uuid"
"X-Kafka-__TypeId__" gets to Kafka as "__typeid__"

Would it be possible to fix it?

**Comment by horkhe** [comment]
Is your application written in Go? If so then it is the Go HTTP client that performs header name normalisation. Try testing with curl. Kafka-Pixy itself does not modify header name case.

**Comment by hansut** [comment]
No, I am using Postman. But I've just tried with SoapUI and curl, and the result is the same.

**Comment by horkhe** [comment]
I looked into the Golang HTTP server implementation and turns out it performs header name normalisation before passing request data to the Kafka-Pixy handler. So it is impossible to preserver the original header case, unless we implement an alternative way to pass Kafka headers that is via request parameters rather than request headers. It should be a small change, but I do not have time to do that right now. If you really need it you can suggest a PR.

**Comment by hansut** [comment]
Thank you for the quick analysis. How can I suggest a PR? From my experience, pull requests are open by the developers who actually implement a change/bugfix, and I actually don't feel to be the right person for implementing this change as I am not familiar with the Go language. 
Anyway it is not so urgent for me, so I suggest to label it as bug or change request and fix it when possible.

**Comment by horkhe** [comment]
I was suggesting you to make the change, but if you are not familiar with the language then it makes sense to just table it for later.
